frontend/pkg/ocm/ocm.go

```diff
@@ -0,0 +1,148 @@
+package ocm
+
+import (
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	v1 "github.com/openshift/api/config/v1"
+
+	"github.com/Azure/ARO-HCP/internal/api"
+	"github.com/Azure/ARO-HCP/internal/api/arm"
+)
+
+const (
+	csCloudProvider    string = "azure"
+	csProductId        string = "aro"
+	csHypershifEnabled bool   = true
+	csMultiAzEnabled   bool   = true
+	csCCSEnabled       bool   = true
+)
+
+// ConvertCStoFrontend converts a CS Cluster object into HCPOpenShiftCluster object
+func ConvertCStoFrontend(cluster *cmv1.Cluster) (*api.HCPOpenShiftCluster, error) {
```

Comment:
Minor optional suggestion: it could be interesting to rename this to _ConvertCStoHCPOpenShiftCluster_ as this could potentially be used in different places than just the frontend, and we don't need to couple here the concept as the function is not coupled with anything about the frontend, it is just called from there.

Reply:
Agree, and another thing to keep in mind is we'll need a similar function for node pools.

Reply:
Updated